templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -488,7 +488,7 @@
      });
 
      function showHideCancelInfo(obj) {
-       if (obj.find(":selected").text() == 'Refunded' || obj.find(":selected").text() == 'Cancelled') {
+       if (obj.find(":selected").val() == '7' || obj.find(":selected").val() == '3') {
```

Comment:
I fully agree, the comparison should be against the ``name`` field of the option value, which -by convention- is not being translated. However, ``value`` was easily available right there... What do you think we should do?